### PR TITLE
Added need help section

### PR DESCRIPTION
**Pull request by Indhumathi040398**
1. Added need help section
2. Created button for need help section

### CODE REVIEW

File: lib/common/theme/color_constants.dart [hunk 1]
```diff
@@ -4,4 +4,7 @@ class ColorConstants {
   static const whiteColor = Color(0xFFFFFFFF);
   static const blackColor = Color(0x00000000);
   static const buttonColor = Color(0xFFF99D4C);
+  static const needHelpColor = Color(0xFF090233);
```
Comment (kingmanimarankv) [inline lib/common/theme/color_constants.dart:7]: Change the variable name to violetColor and format the code.

File: lib/common/widgets/needHelpButton.dart [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:shuttle_whizz/common/theme/color_constants.dart';
+import 'package:shuttle_whizz/constants/string_constants.dart';
+
+class NeedHelpButton extends StatelessWidget {
```
Comment (kingmanimarankv) [inline lib/common/widgets/needHelpButton.dart:6]: Change the name to OutlineButton

File: lib/constants/string_constants.dart [hunk 3]
```diff
@@ -12,5 +12,8 @@ class StringConstants {
   static const allAgesSubTitle ="We have suitable training programs from kids to adult. Enroll now to fast-track your badminton coaching";
   static const membershipsSubTitle ="We offer different membership programs to enhance the hassle free experinence in playing.";
   static const allSkillLevelsSubTitle = "We have suitable training programs from begginner level to Professional level to excel in the game";
+ static const needHelpHeading ="Need help?";
+ static const needHelpSubHeading ="We'd love to help you";
+ static const needHelpButtonText ="Contact us";
```
Comment (kingmanimarankv) [inline lib/constants/string_constants.dart:17]: change the variable name to needHelp, needHelpSubTitle, contactUs

File: lib/screens/widgets/needHelp_section.dart [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:shuttle_whizz/common/widgets/button.dart';
+import 'package:shuttle_whizz/constants/image_constants.dart';
+import 'package:shuttle_whizz/constants/string_constants.dart';
+
+import '../../common/theme/color_constants.dart';
+import '../../common/widgets/needHelpButton.dart';
+
+class NeedHelpSection extends StatelessWidget {
```
Comment (kingmanimarankv) [inline lib/screens/widgets/needHelp_section.dart:10]: Change the class name to HelpSection

File: lib/screens/widgets/needHelp_section.dart [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  const NeedHelpSection({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: ColorConstants.needHelpColor,
+      width: double.infinity,
+      height: 359,
+      child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
+        Text(StringConstants.needHelpHeading,
```
Comment (kingmanimarankv) [inline lib/screens/widgets/needHelp_section.dart:20]: Replace the variable name as mentioned before

File: lib/screens/widgets/needHelp_section.dart [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+            style: GoogleFonts.poppins(
+                color: ColorConstants.whiteColor,
+                fontWeight: FontWeight.w900,
+                fontSize: 50)),
+                SizedBox(height: 8,),
+        Text(StringConstants.needHelpSubHeading,
```
Comment (kingmanimarankv) [inline lib/screens/widgets/needHelp_section.dart:26]: Replace the variable name as mentioned before

File: lib/screens/widgets/needHelp_section.dart [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+            style: GoogleFonts.lato(
+                color: ColorConstants.whiteColor,
+                fontWeight: FontWeight.w300,
+                fontSize: 20)),
+                SizedBox(height:39),
+        NeedHelpButton(),
```
Comment (kingmanimarankv) [inline lib/screens/widgets/needHelp_section.dart:32]: Change the button class name as mentioned before

**Review by kingmanimarankv — CHANGES_REQUESTED** [review]

**Comment by kingmanimarankv** [comment]
Always follow conventions while creating the branches. Branch names should follow lower_snake_case

File: lib/common/widgets/needHelpButton.dart (hunk 2 above)
Comment (kingmanimarankv) [reply lib/common/widgets/needHelpButton.dart:6]: Rename the file name to need_help_button.dart
Always follow naming convention

File: lib/screens/widgets/needHelp_section.dart (hunk 4 above)
Comment (kingmanimarankv) [reply lib/screens/widgets/needHelp_section.dart:10]: Rename the file name to help_section.dart

File: lib/common/theme/color_constants.dart (hunk 1 above)
Comment (Indhumathi040398) [reply lib/common/theme/color_constants.dart:7]: Changed color name to violetColor

File: lib/common/widgets/needHelpButton.dart (hunk 2 above)
Comment (Indhumathi040398) [reply lib/common/widgets/needHelpButton.dart:6]: Changed NeedHelpButton to OutlineButton

File: lib/constants/string_constants.dart (hunk 3 above)
Comment (Indhumathi040398) [reply lib/constants/string_constants.dart:17]: Made changes as requested

File: lib/screens/widgets/needHelp_section.dart (hunk 4 above)
Comment (Indhumathi040398) [reply lib/screens/widgets/needHelp_section.dart:10]: Changed class name as mentioned

File: lib/screens/widgets/needHelp_section.dart (hunk 5 above)
Comment (Indhumathi040398) [reply lib/screens/widgets/needHelp_section.dart:20]: Changes reflected as done before

File: lib/screens/widgets/needHelp_section.dart (hunk 6 above)
Comment (Indhumathi040398) [reply lib/screens/widgets/needHelp_section.dart:26]: Changes reflected as done before

File: lib/screens/widgets/needHelp_section.dart (hunk 7 above)
Comment (Indhumathi040398) [reply lib/screens/widgets/needHelp_section.dart:32]: Changes made

**Comment by Indhumathi040398** [comment]
Fixed all the review comments

**Review by kingmanimarankv — APPROVED** [review]

**Pull request merged**